app/src/app/navtitle/navtitle.component.sass

```diff
@@ -35,6 +37,11 @@ h2
   position: relative
   top: 3px
 
+  @media (max-width: $breakpoint-tablet)
+    top: -3px
+    font-size: small
+    font-weight: bold
```

Comment:
Almost the same here: symbolic font weight are almost deprecated now. Use numbers instead. Beware that Livvic is currently available only in weight 300 and 400. If you want another weight, you must add it in index.html (line 10).